.env.example

```
@@ -1,6 +1,10 @@
 # Required
 NUXT_PUBLIC_SUPABASE_URL="https://example.supabase.co"
 NUXT_PUBLIC_SUPABASE_KEY="<your_publishable_key>"
+NUXT_RESEND_API_KEY=
+NUXT_RESEND_CONTACT_TO=
+NUXT_RESEND_CONTACT_FROM=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NUXT_RESEND_CONTACT_FROM key should go before the NUXT_RESEND_CONTACT_TO key

(UnorderedKey)